spec/features/saml/saml_logout_spec.rb

```diff
@@ -77,7 +77,11 @@
         request = OneLogin::RubySaml::Logoutrequest.new
         visit request.create(settings)
 
+<<<<<<< HEAD
```

Comment:
conflicts

Reply:
Ugh, I must not have checked in these in while I was rebasing.

Reply:
Resolved